### PR TITLE
[Fix] fix triu

### PR DESCRIPTION
**Pull request by AllentDan**
#791

### CODE REVIEW

**Review by grimoire — APPROVED** [review]
LGTM, better rename the rewriter function.

**Review by tpoisonooo — APPROVED** [review]
LGTM.

File: mmdeploy/backend/ncnn/init_plugins.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def get_onnx2ncnn_path() -> str:
 
     if onnx2ncnn_path is None or not os.path.exists(onnx2ncnn_path):
         onnx2ncnn_path = shutil.which('mmdeploy_onnx2ncnn')
+        onnx2ncnn_path = '' if onnx2ncnn_path is None else onnx2ncnn_path
```
[the remark this answers is not in the document]
Comment (AllentDan) [reply mmdeploy/backend/ncnn/init_plugins.py:34]: yes, `shutil.which` returns None sometimes which is not accepted by `osp.exists` in `is_custom_ops_available`

[the remark this answers is not in the document]
Comment (AllentDan) [reply mmdeploy/backend/ncnn/init_plugins.py:34]: AFAK, `''` is the same as `""` for python

**Pull request merged**